geomesa-utils/src/main/resources/org/locationtech/geomesa/geomesa-site.xml.template

```diff
@@ -315,4 +315,12 @@
         <final>false</final>
     </property>
 
+    <property>
+        <name>geomesa.hbase.site.xml.path</name>
```

Comment:
could you add a snippet in the docs about this param?

Reply:
is this a duplicate of https://github.com/locationtech/geomesa/blob/master/geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseDataStoreFactory.scala#L193

Reply:
we may just need better doc?